### PR TITLE
Add Clock support in node

### PR DESCRIPTION
**Pull request by sasha240100**
This pull request is based on issue #8112 
Current usage of `( performance || Date ).now()` breaks node.js with the following error:
```
ReferenceError: performance is not defined
```
I suggest the use of `( typeof performance === 'undefined' ? Date : performance ).now()`. It works in node.js environment as expected.

### CODE REVIEW

File: src/core/Clock.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ Object.assign( Clock.prototype, {
 
 	start: function () {
 
-		this.startTime = ( performance || Date ).now();
+		this.startTime = ( typeof performance === 'undefined' ? Date : performance ).now();
```
Comment (Mugen87) [inline src/core/Clock.js]: Can you please add a comment in this line? Something like // see #10732

**Review by Mugen87 — CHANGES_REQUESTED** [review]

File: src/core/Clock.js (hunk 1 above)
Comment (sasha240100) [inline src/core/Clock.js]: @Mugen87 Will do now

Comment (sasha240100) [inline src/core/Clock.js]: @Mugen87 Done

**Comment by Mugen87** [comment]
I don't now why, but the format of the comment looks strange. Maybe like this? 😉 
```javascript

this.startTime = ( typeof performance === 'undefined' ? Date : performance ).now(); // see #10732
```

**Comment by sasha240100** [comment]
@Mugen87 I added this one first, then looked to other comments to follow your code style and moved to line before, lol. Will update that again now

**Comment by sasha240100** [comment]
@Mugen87 I reimplemented that commit. See now

**Comment by sasha240100** [comment]
@mrdoob @Mugen87 Anything else?

**Comment by Mugen87** [comment]
Looks good!

**Comment by sasha240100** [comment]
@Mugen87 @mrdoob Will it be merged?

**Comment by sasha240100** [comment]
@Mugen87 @mrdoob is it still of current interest?

**Comment by sasha240100** [comment]
Or should i close it?

**Comment by Mugen87** [comment]
It's still a valid PR but I'm unable to merge. Let's wait what @mrdoob says...

**Comment by mrdoob** [comment]
Looks good indeed! (Sorry for the delay!)

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**